packages/appstore/src/components/trustpilot-star-rating/trustpilot-star-rating.tsx

```diff
@@ -1,25 +1,35 @@
 import React from 'react';
 import { Icon } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
 import './trustpilot-star-rating.scss';
 
-const TrustpilotStarRating = ({ score }: { score: number }) => {
+const TrustpilotStarRating = observer(({ score }: { score: number }) => {
+    const { common } = useStore();
+    const { current_language } = common;
+    const html = document.getElementsByTagName('html');
+    const [is_rtl, setIsRtl] = React.useState(html?.[0]?.getAttribute('dir') === 'rtl');
```

Comment:
Can we use  `useIsRtl` hook we have here?

Reply:
@mitra-deriv great, I didn't know we have this hook. I'll update the PR very soon

Reply:
@mitra-deriv something wrong with this hook, application crashed. I can't use it for now

<img width="1351" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/20d132ff-d555-47e4-81b0-eba7c9169997">

Reply:
I think because the appstore's package.json doesn't include  "react-i18next". Should solve the issue once it's being installed inside the package @sergei-deriv @mitra-deriv

Reply:
@suisin-deriv thank you, I'll try to do this right now

Reply:
@suisin-deriv I still see the same error. This hook doesn't work for me. I've added reat-i18next to the package.json and built a project but it doesn't work for me

<img width="304" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/cc4be4e6-926e-47fa-9830-238c0324446c">

<img width="700" alt="image" src="https://github.com/binary-com/deriv-app/assets/120570511/931c0ef9-eb85-4cec-a9af-e10e22c2070a">